util/users.py

```diff
@@ -97,6 +98,8 @@ async def check_user_password_matches(login_form: user_models.UserLoginForm,
 
 
 # fixme: change this to return a Token once we have made the class
-async def get_auth_token_from_user_data(_user: user_models.User) -> str:
-    login_response = 'a jwt!'
+async def get_auth_token_from_user_data(user: user_models.User) -> str:
+    user_id = user.get_id()
+    payload_dict = {'user_id': user_id}
+    login_response = Token.get_enc_token_str_from_dict(payload_dict)
     return login_response
```

Comment:
calling this `login_response` no longer makes sense (before it was being cast into a `UserLoginResponse` object).

Something more obvious like `encoded_jwt_str` or similar would be better.